pkg/cmd/pipelineresource/resource_testUtil.go

```diff
@@ -56,7 +56,7 @@ func (pt *promptTest) runTest(t *testing.T, procedure func(*expect.Console) erro
 
 	// Multiplex output to a buffer as well for the raw bytes.
 	buf := new(bytes.Buffer)
-	c, state, err := vt10x.NewVT10XConsole(expect.WithStdout(buf), expect.WithDefaultTimeout(1*time.Minute))
+	c, state, err := vt10x.NewVT10XConsole(expect.WithStdout(buf), expect.WithDefaultTimeout(10*time.Minute))
```

Comment:
10min is quite a lot (especially in unit tests) and is also the default value for the whole `go test` timeout (aka all unit tests). Any reason to go that high ?

Reply:
I wonder if we don't overcomplicate things on the unit-test then… I feel we are testing our dependency here then (survey)